tests/packagedcode/data/docker/psql.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM postgres:15.3

```

Comment:
Could you also add the source of this, nice that you added a comment on the other docker files about their source.